app/src/main/java/org/sil/gatherwords/EntryActivity.java

```diff
@@ -161,7 +161,7 @@ private void onRecord(boolean start) {
     }
 
     private void startRecording() {
-        mAudioFile = getNewDataFile(".3gp");
+        mAudioFile = Util.getNewDataFile(this, ".3gp");
```

Comment:
What reason is there to update the audioFile in this context (specifically, tinkering with the image display?

Reply:
I pulled `getNewDataFile()` out of `EntryActivity` and into `Util`, making it necessary to update this usage of it.